neuromation/api/jobs.py

```diff
@@ -160,44 +163,51 @@ def __init__(self, core: _Core, config: _Config, parse: Parser) -> None:
         name: str = "",
         owners: Iterable[str] = (),
     ) -> List[JobDescription]:
-        url = URL(f"jobs")
+        url = self._config.api_url / "jobs"
         params: MultiDict[str] = MultiDict()
         for status in statuses:
             params.add("status", status.value)
         if name:
             params.add("name", name)
         for owner in owners:
             params.add("owner", owner)
-        async with self._core.request("GET", url, params=params) as resp:
+        params["cluster_name"] = self._config.cluster_name
```

Comment:
should we consider making this optional? `neuro ps` renders jobs' cluster names already if I'm not mistaken.

Reply:
Thanks for the excellent question!
The PR selects `cluster['name']` if the name is specified, otherwise `'default'` is used for old deployments.
Is `'default'` good enough or it will be rejected by trafarets on servers?

Reply:
`'default'` is a good transitory option.
on `POST /api/v1/jobs` we validate the passed cluster name (if any, otherwise we fall back to `default`) against the clusters available to the user.
on `GET /api/v1/jobs` we just make sure that `cluster_name` is a valid name.

there's a UX part of the question: a user performs the following operations:
```
neuro run ...
neuro config switch-cluster ...
neuro run ...
```
should we output both jobs on
```
neuro ps
```
or only the ones running on the active cluster?

Reply:
I have no idea. Now, only the current cluster's jobs are shown.  
Like in Kubernetes, I see resources from the selected context only.
I suggest keep it until people complain.